CHANGELOG.md

```diff
@@ -48,13 +48,20 @@
 
 #### Added
 
+- A new gated feature `Gateway` has been added. This feature gate enables
+  support for Kubernetes [Gateway APIs][gwapis] in the KIC including limited
+  support for the `HTTPRoute` API.
+  [Gateway Milestone 1][gwm1]
```

Comment:
```suggestion
- Support for Kubernetes [Gateway APIs][gwapis] is now available [by enabling 
  the `Gateway` feature gate](https://docs.konghq.com/kubernetes-ingress-controller/2.2.x/guides/using-gateway-api/).
  This is an alpha feature, with limited support for the `HTTPRoute` API.
  [Gateway Milestone 1][gwm1]
```

Reply:
Changelog also needs a date and the compare link at the bottom; we probably want the date tomorrow to give time for docs team review.

Reply:
1bf3bbb7b2e1f0c7b8dc771961b61ccf6666dd43 now includes both changes :vulcan_salute: